Decred Wallet/Features/Settings/CertificateViewController.swift

```diff
@@ -11,12 +11,14 @@ import UIKit
 class CertificateViewController: UIViewController {
     
     @IBOutlet weak var certificate: UITextView!
+    @IBOutlet weak var certificateDesc: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
         self.navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(cancel))
-        self.navigationItem.title = "Certificate"
+        self.navigationItem.title = LocalizedStrings.certificate
         self.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .save, target: self, action: #selector(save))
-        // Do any additional setup after loading the view.
+        
+        self.certificateDesc.text = "\(LocalizedStrings.certificate):"
```

Comment:
This can use the localizable string key in IB. The colon at the end is not that compulsory to require this additional code.